gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -172,7 +175,7 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
                        **kwargs) -> grpc.Channel:
         """Create and return a gRPC channel object.
         Args:
-            address (Optionsl[str]): The host for the channel to use.
+            address (Optional[str]): The host for the channel to use.
```

Comment:
Why was this changed (looks unrelated to your changes)?

Reply:
Typo

Reply:
I approve of fixing typos on the fly!